skiko/src/nativeJsMain/cpp/Font.cc

```diff
@@ -303,42 +303,25 @@ SKIKO_EXPORT KNativePointer org_jetbrains_skia_Font__1nGetPath
 }
 
 
-SKIKO_EXPORT KInteropPointerArray org_jetbrains_skia_Font__1nGetPaths
-  (KNativePointer ptr, KShort* glyphsArr) {
-    TODO("implement org_jetbrains_skia_Font__1nGetPaths");
-}
-
-#if 0
-SKIKO_EXPORT KInteropPointerArray org_jetbrains_skia_Font__1nGetPaths
-  (KNativePointer ptr, KShort* glyphsArr) {
+SKIKO_EXPORT KNativePointer org_jetbrains_skia_Font__1nGetPaths
+  (KNativePointer ptr, KShort* glyphs, KInt count) {
     SkFont* instance = reinterpret_cast<SkFont*>(ptr);
-    int count = env->GetArrayLength(glyphsArr);
-    KShort* glyphs = env->GetShortArrayElements(glyphsArr, nullptr);
 
     struct Ctx {
```

Comment:
Well, `Ctx` maybe not the best name possible here.  Why not `std::unique_ptr<std::vector<KNativePointer>>` and `release` at the end?